spec/fixtures/copy_candidate_response.json

```diff
@@ -1,6 +1,6 @@
 {
   "candidate": {
     "id": "123456",
-    "url": "https://www.workable.com/spi/v3/accounts/subdomain/candidates/123456"
+    "url": "https://subdomain.workable.com/spi/v3/accounts/subdomain/candidates/123456"
```

Comment:
This also doesn't match the url specified in the docs at https://workable.readme.io/docs/job-candidates-show